### DISCUSSION TITLE
Anyway to capture rtmp stream in Streamlink 6.1.0?

### DISCUSSION THREAD

**Discussion by haskermiller** [streamlink/streamlink#5520] [Q&A] [answered]
### Description

I am still running Streamlink 2.1.2, and I wrote some custom plugins for it.
Today I want to upgrade all custom plugins to work with Streamlink 6.1.0, but I just found out that RTMP support has being dropped long time ago.

But the website and app I want to capture are still using only RTMP, so I need Streamlink 6.1.0 able to capture RTMP Stream.

What's the fastest way to add RTMP support for Streamlink 6.1.0?
Is there an existing repository already done this?
Or I need to do it by myself? And where should I start? Download 2.4.0 and modify code to make it compatible with 6.1.0?

**Comment by bastimeyer** [comment, marked as answer]
RTMP has been removed in 3.0.0:
https://streamlink.github.io/changelog.html#streamlink-3-0-0-2021-11-17

You can have a look at reverting the relevant changes from #4169, but there will be no support here. You are on your own. Good luck.
